package.json

```diff
@@ -7,7 +7,7 @@
     "node": "10.11.0"
   },
   "scripts": {
-    "test": "mocha test --require babel-core/register --require babel-polyfill --exit --timeout 20000",
+    "test": "mocha test/**/** --require babel-core/register --require babel-polyfill --exit --timeout 20000",
```

Comment:
why are we using both timeout and exit flags? I think one of either will be sufficient. Please correct me if I'm worng.